Core/src/org/sleuthkit/autopsy/modules/hashdatabase/HashDbManager.java

```diff
@@ -301,12 +344,12 @@ public void propertyChange(PropertyChangeEvent event) {
      *
      * @throws HashDbManagerException
      */
-    public synchronized void removeHashDatabase(HashDb hashDb) throws HashDbManagerException {
+    public synchronized void removeHashDatabase(HashDatabase hashDb) throws HashDbManagerException {
         this.removeHashDatabaseNoSave(hashDb);
         this.save();
     }
     
-    public synchronized void removeHashDatabaseNoSave(HashDb hashDb) throws HashDbManagerException {
```

Comment:
Are these new enough public methods that we can change them? Or do we need to add deprecated methods that take a HashDb argument?

Reply:
That one got replaced with:
public synchronized void removeHashDatabaseNoSave(HashDatabase hashDatabase) throws HashDbManagerException {
I think that will be invisible to anyone using it - HashDb implements HashDatabase, so I can't make a deprecated version anyway.

Reply:
HashDb implementing HashDatabase was detail I missed.  All makes sense now.